waltz-ng/client/applications/app-view.js

```diff
@@ -131,17 +131,10 @@ function controller($q,
 
     function loadSecondWave() {
         const promises = [
-            measurableCategoryStore
-                .findAll()
-                .then(cs => vm.measurableCategories = cs),
 
-            measurableRatingStore
-                .findByAppSelector({ entityReference, scope: 'EXACT' })
-                .then(rs => vm.ratings = rs),
-
-            measurableStore
-                .findMeasurablesRelatedToPath(entityReference)
-                .then(ms => vm.measurables = ms),
+            entityStatisticStore
+                .findStatsForEntity(entityReference)
+                .then(stats => vm.entityStatistics = stats),
```

Comment:
Why is this change re-introducing ent-stats and surveys at the app-view level ?

Reply:
looks like a dodgy merge, taking another look